### PR TITLE
Select a region using a shape file post

### PR DESCRIPTION
**Pull request by paolap**
Hi, here's a draft on post for the solution I used recently to help Kim and Catherine.
I have to confirm the name for the region defined by the shape file.

### CODE REVIEW

**Comment by HoWol76** [comment]
I like it. I found two typos in the second block: An extra `n` in `geonpandas` and a missing `r` in `rastestats`.

A little bit further down, you talk about having to 
> reproject the shapefile

I would probably say something like "adjust the projection of the shapefile" (I think afterwards you can still use the `reproject` in the comments in the code block.)

**Comment by HoWol76** [comment]
I just noticed that you already fixed the typos. Sorry, I clicked on the 'ReviewNB link which opened the old version.

File: posts/2023-07-05-select-region-shapefile.ipynb [hunk 1]
```diff
@@ -0,0 +1,2641 @@
...
+    "gdf = gpd.read_file(shapefile)\n",
+    "gdf.geometry"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e65f3927-9b6c-439a-8147-18e2061bf149",
+   "metadata": {},
+   "source": [
+    "There is no `crs` (Coordinate reference System) defined for this shapefile so I'm using the array `crs` which is a standard one. If the `crs` for the shapefile and the data array are different you have to reproject the shapefile."
```
Comment (HoWol76) [inline posts/2023-07-05-select-region-shapefile.ipynb]: I would change
"reproject the shapefile" 
into
"change the projection of the shapefile"

Comment (paolap) [reply posts/2023-07-05-select-region-shapefile.ipynb]: thanks I've followed the suggestion

**Pull request merged**